README.md

```diff
@@ -46,6 +46,21 @@ cd csharp\examples\simple
 dotnet run
 ```
 
+## Connecting to GCP GKE from local machine
```

Comment:
These instructions are generic to any cloud provider, so it likely should be:

`Connecting to a cluster from a local machine`

or somesuch.